src/renderer/components/cluster-icon/cluster-icon.tsx

```diff
@@ -47,9 +45,29 @@ export class ClusterIcon extends React.Component<Props> {
     ]);
   }
 
+  get iconString() {
```

Comment:
This method is asking to be tested. Maybe moved into some separate place to make unit test for this?